### PR TITLE
switch to guid to avoid mismatch between k8s and cf

### PR DESCRIPTION
**Pull request by MHBauer**

### CODE REVIEW

File: k8s/service_controller/server/k8s/storage.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func (kss *K8sServiceStorage) AddBroker(broker *model.ServiceBroker, catalog *mo
 	// versions:
 	// - name: v1alpha1
 	ksb := NewK8sSB()
-	ksb.Metadata = Meta{Name: broker.Name}
+	ksb.Metadata = Meta{Name: broker.GUID}
```
Comment (duglin) [inline k8s/service_controller/server/k8s/storage.go]: I think this is right but just want to make sure, where is this info used? If its used internally and its required to be a constant even in the face of a SB rename then I agree.

Comment (MHBauer) [inline k8s/service_controller/server/k8s/storage.go]: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#metadata

Comment (duglin) [inline k8s/service_controller/server/k8s/storage.go]: any reason you're not using Name for Name and using GUID on the uid field?

Comment (MHBauer) [inline k8s/service_controller/server/k8s/storage.go]: uid cannot be set. k8s internally generated.

Comment (MHBauer) [inline k8s/service_controller/server/k8s/storage.go]: name is the 'unique identifier in a namespace'

**Comment by duglin** [comment]
LGTM

**Pull request merged**